src/app/core/constructors/kit/previewKit.constructor.js

```diff
@@ -16,9 +16,10 @@
 
         this.dropdownOptions = [];
 
-        if (!object.kit || object.kit.id === 2 || object.kit.id === 3){
+        if (!object.kit_id || object.kit_id === 2 || object.kit_id === 3){
```

Comment:
maybe we should configure the json render of rails to give us `camelCase` keys and not `snake_case`

Reply:
Well I think it point is to get this from the object after it's parsed but I leave this for another time.